great-idea/css/index.css

```
@@ -63,4 +63,202 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+#menu {
+    padding: 60px 20px 0px 50px;
+}
+#menu a{
+    color:#b7b7b7;
+    text-decoration: none;  
+}
+#menu a:hover{
+    color:red;
+    font-size: 20px;
+    transition: font-size 0.3s ease;    
+}
+#menu ul li{
+    float: left;
+    padding: 15px 40px;
+    height: 30px;
+}
+.logo {
+    float: right;
+    padding: 0px 100px 0px 0px;
+    width: 350px;
+}
+button{
+    background: white;
+    border: 1px solid black;
+    padding: 8px 55px;
+    position: absolute;
+    top: auto;
+    left: 255px;
+    bottom: auto;
+    right: 300px;
+}
+
+.code {
+    padding-bottom: 200px;
+    position: absolute;
+    top: 160px;
+    left: auto;
+    bottom: auto;
+    right: 120px;
+    width: 479px;
+}
+ 
+#one{
+    font-size: 550%;
+    text-align: left;
+    padding: 180px 10px 0px 160px;
+}
+#two{
+    font-size: 550%;
+    text-align: left;
+    padding: 0px 10px 0px 300px;
+}
+#three{
+    font-size: 550%;
+    text-align: left;
+    padding: 0px 10px 0px 220px;
+}
+
+.top-content {
+    position: relative;
+    top: 200px;
+    left: auto;
+    bottom: auto;
+    right: auto;
+    height: 300px;
+    width: 600px;
+    border-top: 1px solid black;
+    padding: 50px 30px;
+}
+#left1 {
+float: left;
+}
+#right1 {
+float: right;
+}
+.middle-img {
+padding: 80px 0px 0px 30px;
+width: 1176px;
+}
+.bottom-content {
+    position: relative;
+    top: 0px;
+    left: auto;
+    bottom: auto;
+    right: auto;
+    height: 300px;
+    width: 400px;
+    padding: 20px 20px 0 20px;
+}
+#left {
+float: left;
+}
+#middle {
+position: relative;
+top: auto;
+left: 400px;
+bottom: 280px;
+right: auto;
+}
+#right {
+float: right;
+position: relative;
+top: auto;
+left: auto;
+bottom: 300px;
+right: auto;
+}
+.contact p {
+    padding: 20px 0px 0px 0px;
+}
+
+.contact {
+    padding-left: 20px;
+}
+
+footer {
+    text-align: center;
+    padding: 80px 0px 100px 400px;
+}
+.topPortion {
+    width: 100%;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-around;
+    padding-bottom: 40px;
+}
+.topPortion h3{
+    padding-top: 40px;
+}
+.textBoxes{
+    padding-top: 40px;
+    border-top: 1px solid black;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    align-content: space-around;
+    padding-bottom: 40px;
+}
+#A1{
```

Isaiah, right here in this section you created various Ids for each of the containers with the same styling. If you are going to use the same styling in the same places you can do it with one class with all the styling and apply that to each container in the html. Remember classes are recyclable and can be used in many containers at the same time
```suggestion
.cards{
 border: 1px solid black;
    background: whitesmoke;
    width: 600px;
    height: 300px;
    margin-bottom: 20px;
}
```